submissions/franqly/franqly.md

```diff
@@ -0,0 +1,99 @@
+
+`Not recently, no.  I tried Pyandons suite a few months ago, got overwhelmed and shelved it.  The base game is definitely "tainted" for me at this point, but I think that is true for most players that have 2k+ hours in the game.  We can never get that first play-through experience again.  I play other games to relax.`
+
+*What other games do you enjoy playing? I saw you're currently into Satisfactory?*
+
+`Yeah, Dyson currently, and satisfactory are both great and i highly recommend both for those that enjoy factorio.  I have played a very wide range of genres in the past, too many to list.  In recent years I've really narrowed my focus to strategy games.  Eu4, slay the spire, sc2, and dominion are also recent games I've played.  They address different itches.  I played copious amounts of WoW and later LoL.  Too much time, lol.`
+
+*How do you find the Factorio speedrunning community? It seems that AntiElitz and Nefrums has created a community with great sportmanship, where runners are competivive but still celebrates others achievements.*
+
+`As i mentioned before, i had watched anti stream off and on for a fews years before joining. I think our community is great!  Good mix of personality and backgrounds with a spirit of cooperation.  It's not overly competitive, but enough so it encourage growth.  I'm not personally super intwined in goings on, but i do enjoy popping in for some discussion once in a while.`
+
+*Do you like watching other streamers/lets-players? If so, who are you following most at the moment? How about anyone else participating in the SGDQ?*
+
+`I used to watch on Twitch a lot more than i do now.  When I do, I've been watching Arumba, Zoast, Cheese, and a few others.  I'm most excited for super metroid at GDQ, oldie but a goodie.`
+
+*Do you have any other hobbies/activities/profession that you like doing outside of the videogame world?*
+
+`I enjoy spending time with my wife.  We watch shows, go on local walks or drive out to scenic hiking spots.  Nothing too crazy, my life is pretty chill atm.  Looking forward to having kids, which we're delaying until we have a house.  Saving as much as we can.  Housing market is ridiculous though.`
+
+## The Event with a Good Cause
+
+We are all very excited to see Franqly take part in this great event where he will be representing Factorio! He has worked very hard and spent ~~countless~~ many hours achieving this amazing goal. And hopefully this will be the first inclusion of many of our favourite factory building game in mainstream events! 
+
+We ask you all join and support Franqly with us! His will be joined in his stream by AntiElitz and Thue, and also [XenoCyber](https://www.twitch.tv/xenocyber) who will guide us through what Franqly is doing. His run will happen on the [GDQ Twitch Channel](https://www.twitch.tv/gamesdonequick) on Friday, 9 July, at 2:18 PM UTC time. 
```

Comment:
Bit weird to link Xeno's twitch here and not Anti's. He's the only one of these three people that streams often :p